src/components/oculus-touch-controls.js

```diff
@@ -139,26 +174,38 @@ module.exports.Component = registerComponent('oculus-touch-controls', {
     this.el.setAttribute('obj-model', {obj: objUrl, mtl: mtlUrl});
   },
 
-  injectTrackedControls: function () {
+  injectTrackedControls: function (gamepad) {
     var el = this.el;
     var data = this.data;
     var isRightHand = data.hand === 'right';
-
-    // since each hand is named differently, avoid enumeration
+    // Inject with specific gamepad id, if provided. This works around a temporary issue
+    // where Chromium uses `Oculus Touch (Right)` but Firefox uses `Oculus Touch (right)`.
     el.setAttribute('tracked-controls', {
```

Comment:
yeah, I have a similar not-just-tracked-controls set for aframe-arcade, but that's beyond this PR's scope, and last we discussed, core team preference was to keep non-tracked controllers (gearvr, daydream, oculus remote, xbox controller, etc) out of core.  maybe file new issue to support controllers included with VR equipment including 0DOF (emulated pose and orientation) and 3DOF (emulated pose, true orientation)?